opentelemetry-appender-log/Cargo.toml

```diff
@@ -34,3 +34,6 @@ opentelemetry-stdout = { path = "../opentelemetry-stdout", features = ["logs"] }
 log = { workspace = true, features = ["kv_serde"] }
 tokio = { workspace = true }
 serde = { workspace = true, features = ["std", "derive"] }
+
+[lints]
+workspace = true
```

Comment:
Just curious, since clippy lints are already defined at the workspace level, won't that get inherited here automatically? Or do we also need `workspace = true` ?

Reply:
Sadly not, it is similar to defining workspace dependency versions. Each crate needs to opt-in.https://doc.rust-lang.org/cargo/reference/workspaces.html#the-lints-table